pref/qho.jl

```diff
@@ -0,0 +1,76 @@
+using QuBase
+using QuDynamics
+using BenchmarkLite
+
+type SampleProc{Alg} <: Proc end
+
+type Qexpmv end
```

Comment:
Can't you use `QuDynamics.QuExpmV` directly for `Alg`? That should simplify things a bit.